modules/auxiliary/scanner/smb/smb_ms17_010.rb

```diff
@@ -48,11 +48,10 @@ def run_host(ip)
       if status == "STATUS_INSUFF_SERVER_RESOURCES"
         print_warning("Host is likely VULNERABLE to MS17-010!")
         report_vuln(
-          :host   => rhost,
-          :port   => rport,
-          :proto  => 'tcp',
-          :sname  => 'SMB',
-          :info   => "Vulnerable to MS17-010",
+          host: ip,
+          name: self.name,
+          refs: self.references,
+          info: 'STATUS_INSUFF_SERVER_RESOURCES for FID 0 against IPC$'
```

Comment:
Happy to change this to something else, but this is what made the most sense to me, given that we're already reporting module title and references.